benchmarks/ImageProcessing/CMakeLists.txt

```diff
@@ -117,3 +117,29 @@ target_link_libraries(image-processing-benchmark
   # Link Buddy MLIR DIP Library.
   BuddyLibDIP
   )
+
+  #-------------------------------------------------------------------------------
+# Image Processing Resize Benchmark Target
+#-------------------------------------------------------------------------------
+
+add_executable(image-processing-resize-benchmark
+MainResize.cpp 
+BuddyResize2DBenchmark.cpp
+)
+
+target_include_directories(image-processing-resize-benchmark
+PRIVATE
+${BUDDY_SOURCE_DIR}/benchmarks/ImageProcessing/include/
+)
+
+target_link_directories(image-processing-resize-benchmark
+PRIVATE 
+${BUDDY_MLIR_LIB_DIR}
+)
+
+target_link_libraries(image-processing-resize-benchmark
+GoogleBenchmark
+${OpenCV_LIBS}
+# Link Buddy MLIR DIP Library.
+BuddyLibDIP
+)
```

Comment:
Add an extra whitespace here

Reply:
Do you mean add a whitespace after line 145?